### DISCUSSION TITLE
docker apache2 web server

### DISCUSSION THREAD

**Discussion by kbarbora** [evertramos/nginx-proxy-automation#355] [Q&A]
Hi just wondering if anybody was able to setup a web server using apache2 behind the proxy?
I just want to save some time if anybody did it or if anyone knows how to set it up.

Thanks

**Comment by evertramos** [comment]
Hello @kbarbora yes you can do that, just put the apache container with the port 80 and the virtual host env with your domain on the same network of the proxy.
